Bootstrap on intel throws an error during vm/switch

From a fresh install on VMWare Fusion 12.2.0 (18760249) on an Intel machine, the following error is thrown during the bootstrap phase (`make vm/bootstrap`) using the latest `main` branch code without any change (impersonating mitchellh on the VM ;-))

```
building the system configuration...
error: anonymous function at /nix/store/z2kiwivn50v7ncd4p362g1gjwi2ng2vi-source/lib/mkvm.nix:3:7 called without required argument 'overlays'

       at /nix/store/z2kiwivn50v7ncd4p362g1gjwi2ng2vi-source/flake.nix:44:36:

           43|
           44|     nixosConfigurations.vm-intel = mkVM "vm-intel" rec {
             |                                    ^
           45|       inherit nixpkgs home-manager;
make[1]: *** [vm/switch] Error 1
```

As per the [video](https://www.youtube.com/watch?v=ubDMLoWz76U), I applied the same settings.
The only change I made is in the makefile to disable the step `vm/secrets` during bootstrap.

--- a/flake.nix
+++ b/flake.nix
@@ -42,7 +42,7 @@
     };
 
     nixosConfigurations.vm-intel = mkVM "vm-intel" rec {
-      inherit nixpkgs home-manager;
+      inherit nixpkgs home-manager overlays;
       system = "x86_64-linux";
       user   = "mitchellh";
     };

